src/modules/CRUD/components/Details/FitBoundButton/FitBoundButton.js

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@blueprintjs/core';
+
+import { getBounds } from '../../../services/features';
+
+const FitBoundButton = ({ coordinates, detailsRef, map, t, title }) => {
+  const handleClick = useCallback(() => {
+    const { current: detail } = detailsRef;
+    if (!map && !detail) {
```

Comment:
I think this should be a OR:

```suggestion
    if (!map || !detail) {
```